cypress/e2e/tests/migration/migration-waves/crud.test.ts

```diff
@@ -31,13 +38,8 @@ describe(["@tier1"], "Migration Waves CRUD operations", () => {
         cy.intercept("DELETE", "/hub/migrationwaves*/*").as("deleteWave");
     });
 
-    it("Fails due to MTA-753 | Migration Wave CRUD", function () {
-        const now = new Date();
-        now.setDate(now.getDate() + 1);
-
-        const end = new Date(now.getTime());
-        end.setFullYear(end.getFullYear() + 1);
-
+    // Automates Polarion TC 332
+    it("Migration Wave CRUD", function () {
```

Comment:
For this method , I just noticed that the checksuccessalert is in the test methods.

Let's move all successalert messages to model class.
for create , edit and delete . 

So the test will look like : 

wave.create()
wave.exists()
wave.edit()
wave.delete()...

Reply:
I understand this comment, and that's why so many tests are failing right now.


Let's consider a model (i.e., Applications) that is extensively used in various tests. However, in most cases where application objects are employed, their primary objective is not to evaluate the applications themselves. Instead, these tests use application objects as a means to verify something else

For example, the Migration Waves tests use applications to verify associations, but their primary objective is to ensure the proper functionality of the waves, not the applications

The question is: Should a problem in the Application model, unrelated to its creation (such as a notification text change), trigger a flag in the Migration Wave CRUD test?

To summarize:

If we validate the notification text within the object class method, it will indeed trigger a flag in the CRUD test for that specific object. However, this approach would also introduce issues in all other tests that utilize the same object for different purposes, it can even hide other bugs

On the other hand, if we solely validate the notification text in the CRUD test, it would flag the issue and confirm the existence of a bug. However, it would not prevent other non-related tests from passing, WDYT?

Reply:
Hmm .. I agree that putting validation messages in models break the tests . 
In that case the create() method for any model should just contain steps to create and no validations .
entity.exists() can check if it is added .

Reply:
Perfect, I'll do that, thanks!

Reply:
Solved.